src/shared/units/formatPercent/formatPercentChange.ts

```diff
@@ -0,0 +1,22 @@
+import { formatPercent } from './formatPercent';
+
+export interface PercentChangeInfo {
+  isPositive: boolean;
+  isNegative: boolean;
+  isNonNegative: boolean;
+  isZero: boolean;
+  formatted: string;
+}
+
+export function formatPercentChange(
+  value: number,
+  locale: string
+): PercentChangeInfo {
+  return {
+    isPositive: value > 0,
+    isNonNegative: value >= 0,
+    isNegative: value < 0,
+    isZero: value === 0,
+    formatted: `${formatPercent(value, locale)}%`,
```

Comment:
We tend to format negative values as `${minus}${format(Math.abs(value))}`
May be here, if we show minus with percent value, this is relevant too :)

Reply:
I see that this is just a copy of code from before. Then this is not critical at all

Reply:
As I can see `formatPercent` does what I mentioned:
```

export function formatPercent(value: BigNumber.Value, locale: string) {
  const valueAsNumber = toNumber(value);
  const formatter =
    valueAsNumber < 1
      ? getFormatterTwoDigits(locale)
      : getFormatterOneDigit(locale);
  const sign = valueAsNumber < 0 ? minus : '';
  return `${sign}${formatter.format(Math.abs(valueAsNumber))}`;
}

```